zim/formats/html.py

```diff
@@ -41,8 +41,8 @@ class Dumper(DumperClass):
 	TAGS = {
 		EMPHASIS: ('<i>', '</i>'),
 		STRONG: ('<b>', '</b>'),
-		MARK: ('<u>', '</u>'),
-		STRIKE: ('<s>', '</s>'),
+		MARK: ('<em>', '</em>'),
```

Comment:
This change probably calls for updating HTML template CSS to make `<em/>` more visually prominent to match default zim style.